circuitbreaker.py

```diff
@@ -62,6 +62,7 @@ def call(self, func, *args, **kwargs):
         if self.opened:
             if self.fallback_function:
                 self.fallback_function(*args, **kwargs)
+                return None
```

Comment:
Better return the fallback function return

```python
return self.fallback_function(*args, **kwargs)
```

Reply:
ok, thanks will fix!

Reply:
Fixed